gradle/project.gradle.kts

```diff
@@ -26,6 +26,7 @@ plugins {
     application
     id("com.github.ben-manes.versions") version "0.20.0"
     id("de.undercouch.download") version "3.4.3"
+    id("org.openjfx.javafxplugin") version "0.0.6"
```

Comment:
I commented on the existence of the plugin below, you can reply here instead.

Reply:
https://github.com/openjfx/javafx-gradle-plugin/blob/master/src/main/java/org/openjfx/gradle/tasks/ExecTask.java It must be investigated whether this plugin adds jvm arguments for our project. Having silently modified jvm args would be quite detrimental.

Reply:
Its ok plugin can stay, but can you pls check the args? I hope no --module or anything is put in jvm args.

Reply:
The plugin indeed adds --module arguments. I would rather avoid it in the future, but for now it is fine.